linera-base/src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub mod crypto;
 pub mod data_types;
 mod graphql;
 pub mod identifiers;
+pub mod prometheus_util;
```

Comment:
Could also just be called `prometheus`.

More importantly, `linera_base` is compiled in Wasm and we don't want to include the `prometheus` crate in this case.

Reply:
I think I'm maintaining `prometheus_util` because it seems less confusing in the code, confuses even the compiler a bit also

Reply:
sure, as long as it is not included in Wasm builds